frame/evm/precompile/contracts/bridge/bsc/src/lib.rs

```diff
@@ -104,16 +101,22 @@ where
 							params.account_proof.clone(),
 							storage_proof,
 						);
-						let storage_value = EthereumStorage::<H256>::verify_storage_proof(
+						let storage_value = EthereumStorage::<Vec<u8>>::verify_storage_proof(
 							finalized_header.state_root,
 							&proof,
-						)
-						.map_err(|_| ExitError::Other("verify storage proof failed".into()))?;
-						Ok(storage_value.0.into())
+						);
+						match storage_value {
+							Ok(value) => {
+								return Ok(value.0.clone());
+							}
+							Err(_) => {
+								return Ok(vec![]);
```

Comment:
I am curious about this error here, shouldn't we return Error?  If not, `storage_values?` always OK in my view.

Reply:
we should not return error here. If the storage_key is not exist or the storage value is Zero Value, then we get an error here. We should transfer the error to Zero Value because it's multi verify.

Reply:
> If the storage_key is not exist or the storage value is Zero Value

You mean the `storage_value`, which is the result of `verify_storage_proof` function?